### PR TITLE
bug(runtime): fix dockerize tag tuple typo

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
image

## Modules
- [x] Client

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/core/runtime/model.py [hunk 1]
```diff
@@ -1067,7 +1067,7 @@ def _render_dockerignore() -> None:
             )
 
         def _build(_manifest: t.Dict[str, t.Any]) -> None:
-            _tags = tags or []
+            _tags = list(tags or [])
```
[the remark this answers is not in the document]
Comment (tianweidut) [reply client/starwhale/core/runtime/model.py:1070]: If another place calls this function with a tuple, it still raises an exception.  format a list type is only the `_build` internal implement.

[the remark this answers is not in the document]
Comment (goldenxinxing) [reply client/starwhale/core/runtime/model.py:1070]: yes,i see ,what i point is the file of 'starwhale.core.runtime.cli._dockerize', is it the cli param's type is a Tuple but not a List[str]?
`def _dockerize(
    uri: str,
    tag: t.List[str],
    push: bool,
    platform: t.List[str],
    dry_run: bool,
    use_starwhale_builder: bool,
    reset_qemu_static: bool,
) -> None:
    """[Only Standalone]Starwhale runtime dockerize, only for standalone instance

    URI (str): Starwhale Runtime URI in the standalone instance
    """
    RuntimeTermView(uri).dockerize(
        tag, push, platform, dry_run, use_starwhale_builder, reset_qemu_static
    )`

[the remark this answers is not in the document]
Comment (tianweidut) [reply client/starwhale/core/runtime/model.py:1070]: updated

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**